test/doctool/test-doctool-html.js

```diff
@@ -97,6 +105,16 @@ testData.forEach(function(item) {
           // Assert that the input stripped of all whitespace contains the
           // expected list
           assert.notEqual(actual.indexOf(expected), -1);
+
+          // Testing the insertion of Google Analytics script when
+          // an analytics id is provided. Should not be present by default
+          if (includeAnalytics) {
+            assert.notEqual(actual.indexOf('google-analytics.com'), -1,
```

Comment:
Nit: it may not much of a difference, but everywhere else we prefer `strict` equivalents of assertions.